op-deployer/pkg/deployer/manage/flags.go

```diff
@@ -13,11 +13,26 @@ var (
 		Usage:   "Address to use for as the proxy admin owner. Not compatible with the --workdir flag.",
 		EnvVars: deployer.PrefixEnvVar("PROXY_ADMIN_OWNER"),
 	}
+	PrankFlag = &cli.StringFlag{
+		Name:    "prank-address",
+		Usage:   "Address to use for the prankster.",
+		EnvVars: deployer.PrefixEnvVar("PRANK_ADDRESS"),
+	}
```

Comment:
Can you add any information or rename this flag to indicate what address should be provided here?